internal/manager/manager.go

```diff
@@ -102,14 +102,14 @@ func newMgr(logger logr.Logger, opts *Options, isController, isReconciler bool)
 	}
 	mgrOpts.Cache.DefaultFieldSelector = fieldSelector
 
-	podLabelSelector := labels.SelectorFromSet(labels.Set{ManagerLabelKey: ManagerLabelValue})
-
 	if isController {
-		// We do not honor the configured label selector, because these pods will only ever have labels set by Eno.
-		// But we _do_ honor the field selector since it may reduce the namespace scope, etc.
+		// Only cache pods in the synthesizer pod namespace and owned by this controller
 		mgrOpts.Cache.ByObject[&corev1.Pod{}] = cache.ByObject{
-			Label: podLabelSelector,
-			Field: fieldSelector,
+			Namespaces: map[string]cache.Config{
```

Comment:
Now that synthesizer pods are in the controller's namespace, it doesn't really make sense to filter the cache like we currently are. We can calculate the ns/label selectors without requiring any other configuration.